src/BrazeEndOfArticleComponent.ts

```diff
@@ -34,6 +38,7 @@ const END_OF_ARTICLE_MAPPINGS: ComponentMapping<CommonEndOfArticleComponentProps
     [UK_NEWSLETTER_EPIC_NAME]: UKNewsletterEpic,
     [EPIC_NAME]: Epic,
     [EPIC_WITH_HEADER_IMAGE_NAME]: EpicWithSpecialHeader,
+    [DTO_NEWSLETTER_EPIC_NAME]: DownToEarthNewsletterEpic,
```

Comment:
```suggestion
    [DTE_NEWSLETTER_EPIC_NAME]: DownToEarthNewsletterEpic,
```

Should this be DTE instead of DTO?